<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signin</class>
 <widget class="QWidget" name="signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Signin</string>
   </property>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="ststusLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
